cmd/kyverno/main.go

```diff
@@ -87,7 +90,8 @@ func main() {
 	}
 
 	klog.InitFlags(nil)
-	log.SetLogger(klogr.New())
+	log.SetLogger(klogr.NewWithOptions(klogr.WithFormat(klogr.FormatKlog)))
```

Comment:
You call `SetLogger` here and below, it should  be done only once.

Reply:
👍

Reply:
Please fix this.